backend/src/packages/orders/order.entity.ts

```diff
@@ -22,12 +29,21 @@ class OrderEntity implements IEntity {
 
   private businessId: OrderEntityT['businessId'];
 
-  private driverId: OrderEntityT['driverId'];
-
   private customerName: OrderEntityT['customerName'] | null;
 
   private customerPhone: OrderEntityT['customerPhone'] | null;
 
+  private shift: {
+    id: ShiftEntity['id'];
+  };
```

Comment:
Maybe it will be better to use Pick<ShiftEntity,'id'> instead?

```suggestion
  private shift: Pick<ShiftEntity,'id'>;
```